docs/source/sysadmin_guide/grafana_configuration.rst

```diff
@@ -0,0 +1,194 @@
+   the server running the InfluxDB Docker container. Under "InfluxDB Details",
+   enter the INFLUXDB_USER user and INFLUXDB_USER_PASSWORD password used in the
+   InfluxDB section, then click "Save & Test".
+
+   .. note::
+
+      Sawtooth 1.0.* requires the dashboard from the sawtooth 1-0 branch, while
+      sawtooth 1.1.* can use the dashboard included in the Grafana Docker
+      container from master.
+
+   For Sawtooth 1.0.* only, import the 1.0 dashboard. (Sawtooth 1.1.* can use
+   the dashboard included in the Grafana Docker container.)
+
+   a. Get the 1.0 dashboard from either
+   sawtooth-core/docker/grafana/dashboards/sawtooth_performance.json in the 1-0
+   branch or download from GitHub directly at `sawtooth_performance.json
+   <https://raw.githubusercontent.com/hyperledger/sawtooth-core/1-0/docker/grafana/dashboards/sawtooth_performance.json>`_.
+
+   b. To import the 1.0 dashboard, click Grafana spiral logo on top left of
+   page and mouse over "Dashboards", click "Import" then click "Upload .json
+   file" and navigate to the path where sawtooth_performance.json was saved.
+   Next select "metrics" in the drop-down menu and click "Import".
+
+Sawtooth Configuration
+======================
+
+Sawtooth metrics are reported by the validator process and the configuration
+lives in ``/etc/sawtooth/validator.toml``. You can copy the template from
+``/etc/sawtooth/validator.toml.example`` if the file doesn't exist yet.
+
+#. Fill in the following values with the configurations used above.
+   ``opentsdb_url`` is the IP / FQDN:port to the InfluxDB instance,
+   ``opentsdb_db`` is the value of INFLUXDB_DB, then fill in the INFLUXDB_USER
+   for ``opentsdb_username`` and INFLUXDB_USER_PASSWORD for
+   ``opentsdb_password`` created above.
+
+   .. code-block:: console
+
+     # The host and port for Open TSDB database used for metrics
+     opentsdb_url = "http://{host}:3000"
+
+     # The name of the database used for storing metrics
+     opentsdb_db = "metrics"
+
+     opentsdb_username = "lrdata"
+
+     opentsdb_password = "test"
+
+   .. note::
+      Be sure to replace {host} with the IP or Fully Qualified Domain Name
+      (FQDN) of the system running the InfluxDB Docker container.
+
+#. Restart sawtooth-validator for these changes to take effect.
+
+   If started via systemd:
+
+   .. code-block:: console
+
+     $ sudo systemctl restart sawtooth-validator
+
+Telegraf Configuration
+=======================
+
+Telegraf is used on the sawtooth validator nodes for sending OS and Hardware
+metrics to InfluxDB.
+
+#. Install Telegraf from the InfluxData repository.
+
+   .. code-block:: console
+
+     $ curl -sL https://repos.influxdata.com/influxdb.key | sudo apt-key add -
+     $ sudo apt-add-repository "deb https://repos.influxdata.com/ubuntu xenial stable"
+     $ sudo apt-get update
+     $ sudo apt-get install telegraf
+
+#. To configure Telegraf, edit /etc/telegraf/telegraf.conf
```

Comment:
Add a period at the end of this sentence.  Also, the doc convention is to use double-backticks for all file names, paths, and command names.